internal/nodes/avi_model_evh_nodes.go

```diff
@@ -1160,13 +1160,19 @@ func (o *AviObjectGraph) BuildTlsCertNodeForEvh(svcLister *objects.SvcLister, tl
 		secretNS = namespace
 	}
 
+	re := regexp.MustCompile(fmt.Sprintf(`^%s.*`, lib.DummmySecretK8s))
```

Comment:
please consider moving this to a function since the logic to figure out the secret name is the same for evh and sni.